From 5ebfc62240f1c40683a9db177cbe6517a3c67bf9 Mon Sep 17 00:00:00 2001
From: "Sicelo A. Mhlongo" <absicsz@gmail.com>
Date: Thu, 10 Apr 2025 14:24:14 +0200
Subject: [PATCH 8/8] meson.build: Downgrade gudev version for daedalus

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 4f0aa21..2f01d18 100644
--- a/meson.build
+++ b/meson.build
@@ -79,7 +79,7 @@ gobject_introspection = dependency('gobject-introspection-1.0', required: get_op
 
 
 if os_backend == 'linux'
-  gudev_dep = dependency('gudev-1.0', version: '>= 238')
+  gudev_dep = dependency('gudev-1.0', version: '>= 237')
   idevice_dep = dependency('libimobiledevice-1.0',
                            version : '>= 0.9.7',
                            required : get_option('idevice'))
-- 
2.39.5

